cocos/gfx/base/define.ts

```diff
@@ -1012,42 +1012,19 @@ export class BindingMappingInfo {
     }
 }
 
-export class SystemWindowInfo {
-    declare private _token: never; // to make sure all usages must be an instance of this exact class, not assembled from plain object
-
-    constructor (
-        public title: string = null!,
-        public x: number = 0,
-        public y: number = 0,
-        public width: number = 0,
-        public height: number = 0,
-        public flags: number = 0,
-        public windowHandle: HTMLCanvasElement = null!,
-    ) {}
-
-    public copy (info: Readonly<SystemWindowInfo>) {
-        this.title = info.title;
-        this.x = info.x;
-        this.y = info.y;
-        this.width = info.width;
-        this.height = info.height;
-        this.flags = info.flags;
-        this.windowHandle = info.windowHandle;
-        return this;
-    }
-}
-
 export class SwapchainInfo {
     declare private _token: never; // to make sure all usages must be an instance of this exact class, not assembled from plain object
 
     constructor (
+        public windowId: number = 0,
```

Comment:
This file is generated from c++ using gfx-define-generator. TS windowId is now synchronized with c++ implementation.